app/Platform/Actions/GetUserProgressionStatusCountsAction.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Actions;
+
+use App\Models\PlayerGame;
+use App\Models\System;
+use App\Models\User;
+use App\Platform\Enums\AchievementSetType;
+
+class GetUserProgressionStatusCountsAction
+{
+    private const MIN_ACHIEVEMENTS = 5;
```

Comment:
I realize this is duplicated from the other function, but it's misleading. It should be:
```
private const MIN_ACHIEVEMENTS_FOR_COMPLETION = 6;
```

And the query should use `>=` instead of `>`.

Reply:
It could also just use `PlayerBadge::MINIMUM_ACHIEVEMENTS_COUNT_FOR_MASTERY`, which is I think what the logic is trying to capture.